Sources/Components/ShippingPackageSizeView/Subviews/ShippingPackageSizeButton.swift

```diff
@@ -0,0 +1,139 @@
+import UIKit
+import FinniversKit
+
+class ShippingPackageSizeButton: UIView {
```

Comment:
Since this is a button, it might be a good idea to subclass `UIControl`? Also it probably needs some accessiblity configuration.